on.js

```diff
@@ -581,11 +581,13 @@ define(["./has!dom-addeventlistener?:./aspect", "./_base/kernel", "./sniff"], fu
 						event.rotation = 0;
 						event.scale = 1;
 					}
-					//use event.changedTouches[0].pageX|pageY|screenX|screenY|clientX|clientY|target
-					var firstChangeTouch = event.changedTouches[0];
-					for(var i in firstChangeTouch){ // use for-in, we don't need to have dependency on dojo/_base/lang here
-						delete event[i]; // delete it first to make it mutable
-						event[i] = firstChangeTouch[i];
+					if (window.TouchEvent && originalEvent instanceof TouchEvent) {
```

Comment:
I think the first check should probably be has("touch-events") instead of window.TouchEvent ? Or is https://github.com/dojo/dojo/blob/master/has.js#L110 not specific enough? Or are you looking for a specific touch event that's been fired on the window object? I guess the point is, I prefer these types of tests to be shared wherever possible, so they only need to be updated in one place and are easily reused. Or perhaps this entire condition can be a new has test, unless I'm misunderstanding the check?